### PR TITLE
Added to Conferences/Talks

### PR DESCRIPTION
**Pull request by EggplantPlanter**
Girls in Tech Catalyst Conference description
Issue: https://github.com/cristianoliveira/awesome4girls/issues/99

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,6 +281,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [Ela Conf](http://elaconf.com) *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg" width="30" />
 
     A safe, inclusive tech leadership conference and community for adult (18+) women (cis and trans), trans men, and genderqueer people to connect with and empower one another.
+    
+  - [Girls in Tech Catalyst Conference](http://catalystsf.girlsintech.org/)
+
+    Girls in Tech is an organization compromising of women of any level in their career. They ephasize the importance of empowerment, entrepreneurship, engadgement, and education in the technology industry. With members ranging from high-tech employees to students, they hold yearly conferences to connect and inspire women. Other than building relationships, they offer learning opportunities to broaden your knowledge and develop skills that will have a considerable impact in the tech world and your career.
```
Comment (cristianoliveira) [inline README.md]: We already have the Girls in tech listed. There is no problem in adding the conference separated but would be nice to describe the conference then. I took from their site, how about this:

"Catalyst delivers an experience no other conference can rival. Tedx-style power presentations will have you look at leaders in a whole new light—and forever change the way you approach your life, your career and all the tradeoffs in between. Unique opportunities to connect with other women in tech will not only keep you energized but will make you realize the support system that is at your fingertips."

**Comment by cristianoliveira** [comment]
Hey @EggplantPlanter Thanks for your first contribution! 🎉 🎈 
I left only one concern about this PR. Could you please take a look at it and address? 😄

File: README.md (hunk 1 above)
Comment (EggplantPlanter) [reply README.md]: Whoops, i'm still learning the ropes of using GitHub so I appreciate you helping out. That sounds perfect, i'll go and make the change

Comment (EggplantPlanter) [reply README.md]: ahh also should I include the quotes, sorry >.<

Comment (cristianoliveira) [reply README.md]: No problem :)

You don't need to include the quotes ;)

Comment (cristianoliveira) [reply README.md]: Let me know if you need any help :D

**Comment by cristianoliveira** [comment]
Looks great! Thanks @EggplantPlanter

**Pull request merged**